pkg/build/api/types.go

```diff
@@ -424,7 +424,7 @@ type BuildStrategy struct {
 	CustomStrategy *CustomBuildStrategy
 
 	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
-	// This strategy is experimental.
+	// This strategy is a tech preview.
```

Comment:
"Tech preview" is a state.  You are "in" the tech preview state.

On Aug 11, 2016, at 12:59 PM, Ben Parees <notifications@github.com> wrote:

In pkg/build/api/types.go
<https://github.com/openshift/origin/pull/10288#discussion_r74460917>:

> @@ -424,7 +424,7 @@ type BuildStrategy struct {
>  	CustomStrategy *CustomBuildStrategy
>
>  	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
> -	// This strategy is experimental.
> +	// This strategy is a tech preview.

that's not english...
"this strategy is in tech preview"?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/10288/files/992b79324c91036c236e26f062edabe3e68b60db#r74460917>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ABG_p5ioZR7YHVefl8HRuD85p8DZnTe8ks5qe1T3gaJpZM4JfeX->
.